code/modules/projectiles/guns/flamer/flamer.dm

```diff
@@ -632,7 +632,7 @@
 		if(FIRE_VARIANT_TYPE_B)
 			if(isxeno(M))
 				var/mob/living/carbon/xenomorph/X = M
-				X.armor_deflection?(variant_burn_msg=" You feel the flames weakening your exoskeleton!"):(variant_burn_msg=" You feel the flaming chemicals eating into your body!")
+				X.armor_deflection?(variant_burn_msg=" We feel the flames weakening our exoskeleton!"):(variant_burn_msg=" You feel the flaming chemicals eating into your body!")
```

Comment:
Both messages are for xenos right? you should change both

Reply:
i believe thats for humans

Reply:
after testing this just doesnt get use danywhere for some reason